L77-core-cpp-third-party-identity-support-for-call-credentials.md

```diff
@@ -76,32 +76,47 @@ This doc proposes to extend the access to Google APIS directly with 3p identitie
 
 **Implicit flow**
 
-In core, the underlying credentials retrieval process of `grpc::GoogleDefaultCredentials()` will be extended to accept 3p identity credentials implicitly.
+In core, `grpc::GoogleDefaultCredentials()` follows the ADC (Application Default Credentials) credentials retrieval process. It will be extended to accept 3p identity credentials. More information on ADC and how they work can be found here: https://developers.google.com/identity/protocols/application-default-credentials
```

Comment:
Please change "In core" to "In C++", since this is talking about a C++ API.

Reply:
The comment was addressed in https://github.com/grpc/proposal/pull/221